mesheryctl/cmd/vars.go

```diff
@@ -16,15 +16,19 @@ package cmd
 
 import (
 	"io"
+	"log"
 	"net/http"
 	"os"
+	"os/exec"
 )
 
 const (
-	url                = "http://localhost:9081"
-	fileUrl            = "https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml"
-	mesheryLocalFolder = ".meshery"
-	dockerComposeFile  = mesheryLocalFolder + "/meshery.yaml"
+	url                    = "http://localhost:9081"
+	fileUrl                = "https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml"
+	mesheryLocalFolder     = ".meshery"
+	dockerComposeFile      = mesheryLocalFolder + "/meshery.yaml"
+	dockerComposeBinaryUrl = "https://github.com/docker/compose/releases/download/1.24.1/docker-compose"
```

Comment:
It'd be nice to avoid pinning to a specific docker-compose release #, but attempt latest instead, so that we don't have to maintain `init` as docker-compose move forward.

Can we incorporate a dynamic pull of the latest release #? @girishranganathan, you already have something like this, right? Can that be reused? Something like:

```
git ls-remote --tags https://github.com/docker/compose.git | sed -nE 's#.*refs/tags/(v?[0-9]+(\.[0-9]+)*)$#\1#p' | sort -Vr | head -n 1
```
or 
```
curl -s https://api.github.com/repos/docker/compose/releases/latest | jq -r '.assets[].browser_download_url'
```